sem-reference_5b6968642c7d3a03f89d6bff.md

```diff
@@ -1,2 +1,350 @@
+commands.
+
+## Syntax
+
+The general syntax of the `sem` utility is as follows:
+
+    sem [COMMAND] [RESOURCE] [NAME] [flags]
+
+where `[COMMAND]` is the name of the command as explained in this reference
+page, `[RESOURCE]` is the resource type that interests us, `[NAME]` is the
+actual name of the resource and `[flags]` are optional flags.
+
+Not all `sem` commands require a `[RESOURCE]` value and most `sem` commands do
+not have a `[flag]` part.
+
+Some of the `[flags]` values need an additional argument, as it is the case
+with the `-f` flag, which requires a valid path to a local file.
+
+## Operations
+
+The following list briefly describes the `sem` operations:
+
+* *connect*: The `connect` command is used for connecting to organizations for the first time.
+* *context*: The `context` command is used for adding an organization to your existing ones for the first time.
+* *create*: The `create` command is used for creating new resources.
+* *delete*: The `delete` command is used for deleting existing resources.
+* *describe*: The `describe` command is used for getting information about existing resources.
+* *get*: The `get` command is used for getting the list of an existing type of resource.
+* *help*: The `help` command is used for getting help about `sem` or an existing `sem` command.
+* *init*: The `init` command is used for adding an existing GitHub repository to Semaphore 2.0 for the first time.
+* *version*: The `version` command is used for getting the version of the `sem` utility.
+
+## Resource types
+
+Semaphore 2.0 supports two types of resources: `secrets` buckets and Semaphore
+2.0 projects. Most resource related operations require a resource name.
+
+## Working with organizations
+
+This group of `sem` commands for working with organizations contains the
+`sem connect` and `sem context` commands.
+
+### sem connect
+
+The `sem connect` command allows you to connect to a Semaphore 2.0 organization
+for the first time and requires two command line arguments. The first command
+line argument is the organization name and the second command line argument is
+the authentication token for that organization – the organization must already
+exist. Organizations are created using the web interface of Semaphore 2.0.
+The authentication token depends on the active user.
+
+Once you connect to an organization, you do not need to execute `sem connect`
+for connecting to that organization again – you can connect to any
+organization that you are already a member of using the `sem context` command.
+
+### sem context
+
+The `sem context` command is used for listing the organizations the active
+Semaphore 2.0 user belong to and for changing between organizations.
+
+The `sem context` command can be used with or without any command line
+parameters. If `sem context` is used without any other command line parameters,
+it returns the list of Semaphore 2.0 organizations the active user belongs to.
```

Comment:
This is not true. It only lists organizations that user previously connected to with `sem` CLI.